app/src/components/governance/PageProposals.vue

```diff
@@ -42,13 +68,30 @@
         </TmDataMsg>
       </div>
     </div>
-    <TableProposals v-else :proposals="proposals" />
-    <ModalTutorial
+
+    <template v-else>
+      <LiProposal
+        v-for="proposal in proposals"
+        :key="proposal.id"
+        :proposal="proposal"
+      />
+    </template>
+
+    <ParticipantList
       v-if="
-        showTutorial &&
-        (connection.network === 'cosmos-hub-mainnet' ||
-          connection.network === 'cosmos-hub-testnet')
+        governanceOverview.topVoters && governanceOverview.topVoters.length > 0
       "
+      :title="participantListTitle"
+      :participants="governanceOverview.topVoters"
+    />
+
+    <ProposalDescription
```

Comment:
if this only handles links, the name is confusing

Reply:
@jbibla

Reply:
it also has a description. How to call this component then? `ProposalInfos`?

Reply:
i'm fine with the name - but we could make it more generic i suppose.